webrtc-stats.html

```diff
@@ -1612,6 +1613,21 @@
                 </p>
               </dd>
               <dt>
+                <dfn><code>voiceActivityFlag</code></dfn> of type <span class="idlMemberType"><a>
+                boolean</a></span>
+              </dt>
+              <dd>
+                <p>
+                  Only valid for audio. Whether the last RTP packet played out by this track
+                  contained voice activity or not based on the V bit of the RFC 6464 extension
```

Comment:
It's referenced in an above paragraph already but yeah we should always use [[RFC6464]]. Updated paragraph.